tests/passes/compile-repeat/nested.expect

```diff
@@ -17,14 +17,14 @@ component main(@go go: 1, @clk clk: 1, @reset reset: 1) -> (@done done: 1) {
       r1.write_en = 1'd1;
       write_r1[done] = r1.done;
     }
-    group init_repeat {
+    group init_repeat<"promote_static"=1> {
```

Comment:
Even though compile-repeat runs after static-promotion, I still added the `promote_static` attribute, since maybe one day in the future we will adjust the pass ordering.